tests/storage/test_storage.py

```diff
@@ -436,6 +437,18 @@ def test_fetch_file_contents_using_s3_select_and_expect_output_in_csv_format(sel
             OutputSerialization={"CSV": {"FieldDelimiter": ",", "RecordDelimiter": "\n"}},
         )
 
+    @mock.patch.object(storage.S3FileStore, "_get_boto_object_for_key")
+    def test_get_last_modified(self, get_boto_object_for_key):
+        k = mock.Mock()
+        k.last_modified = datetime.datetime.now()
```

Comment:
🐼 Instead of using `now()` I'd use a concrete date time just to make this test less susceptible to a weird flakey time problem.

Reply:
Oh yes, this is a gotcha I've run into before. This might work on your computer right now, but the next computer might be just fast enough to trigger this as a failure.

Reply:
Great suggestion.  Fixed.